js/console.ts

```diff
@@ -9,7 +9,19 @@ type ConsoleOptions = Partial<{
 }>;
 
 // Default depth of logging nested objects
-const DEFAULT_MAX_DEPTH = 2;
+const DEFAULT_MAX_DEPTH = 4;
+
+const TYPEDARRAY_TYPES = [
```

Comment:
I saw somewhere in another commit of deno use `TypedArray`, which can be exposed by doing `Object.getPrototypeof(Int8Array)`, which all typed arrays extend. It's shorter and provides support for `BigInt64Array` and `BigUint64Array`, which the version of V8 we use supports, and possibly any other typed array that gets implemented in the future.

Reply:
Found it! It's in `util.ts`, we could use this function instead of running a loop for each typed array:

https://github.com/denoland/deno/blob/c427c2df427f477eb1214d8ff3fdfad36e191a6c/js/util.ts#L129-L133

Reply:
Great! Thanks for the suggestion. Will update right away

Reply:
Yeah, that too 👍